### DISCUSSION TITLE
How to run the same job twice?

### DISCUSSION THREAD

**Discussion by Vadorequest** [community/community#25271] [Actions]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/8/e82559ef7ccf2dd10485d6c332ee05a20f667d1d.png" data-download-href="/uploads/short-url/x7ENtlIEAaJ8ZY89s64TMo2dYgl.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3807458/181090006-c4736918-fb20-416a-8ab0-a18f6e1954ea.png" alt="image" data-base62-sha1="x7ENtlIEAaJ8ZY89s64TMo2dYgl" width="690" height="314" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e82559ef7ccf2dd10485d6c332ee05a20f667d1d_2_690x314.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e82559ef7ccf2dd10485d6c332ee05a20f667d1d_2_1035x471.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e82559ef7ccf2dd10485d6c332ee05a20f667d1d_2_1380x628.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e82559ef7ccf2dd10485d6c332ee05a20f667d1d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1678×766 76 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>In the above screenshot, I would like to run the “purge” job several times:</p>
<ul>
<li>After “tag-and-release” job is done</li>
<li>After “await-jsdelivr-cache-update” is done</li>
</ul>
<p>Is there a way to do that without duplicating all the “purge” code? The exact same code needs to run.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Maybe putting the “purge” code into a reusable workflow would do the trick? <a href="https://docs.github.com/en/actions/using-workflows/reusing-workflows" class="inline-onebox">Reusing workflows - GitHub Docs</a></p>

**Comment by miladwyer** [comment]
<aside class="quote group-Advisory-Council quote-modified" data-username="airtower-luna" data-post="2" data-topic="230920" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>Maybe putting the “purge” code into a reusable workflow would do the trick? <a href="https://docs.github.com/en/actions/using-workflows/reusing-workflows" class="inline-onebox" rel="noopener nofollow ugc">Reusing workflows - GitHub Docs</a> <a href="https://iogames.site" rel="noopener nofollow ugc">io games</a></p>
</blockquote>
</aside>
<p>Thanks for the information. It really helped me!</p>
